quill/quill.d.ts

```diff
@@ -1,3 +1,8 @@
+// Type definitions for quill.js
+// Project: https://github.com/quilljs/quill/tree/0.20.1
+// Definitions by: Yao Yujian <https://github.com/yyjhao>
```

Comment:
We generally use `// Definitions by: Asana <https://asana.com>` for the DefinitelyTyped attributions.